insights/tests/client/apps/test_compliance.py

```diff
@@ -132,6 +131,20 @@ def test_run_scan_fail(config, call):
         call.assert_called_with(("oscap xccdf eval --profile ref_id --results " + output_path + ' /nonexistent').encode(), keep_rc=True, env=env)
 
 
+@patch("insights.client.apps.compliance.call", return_value=(0, ''.encode('utf-8')))
+@patch("insights.client.config.InsightsConfig")
+def test_run_scan_missing_profile(config, call):
+    compliance_client = ComplianceClient(config)
+    output_path = '/tmp/oscap_results-ref_id.xml'
+    env = os.environ
+    env.update({'TZ': 'UTC'})
+    assert compliance_client.run_scan('ref_id', None, output_path) is None
+    if six.PY3:
+        call.assert_not_called()
+    else:
+        call.assert_not_called()
```

Comment:
Statements for both `if` branches are the same. Haven't you meant to use a different statement/call?

Reply:
Ah, good point. I'll remove the condition.